tests/unit/disco/components/TestAddon.js

```diff
@@ -195,7 +195,8 @@ describe(__filename, () => {
     });
 
     it('renders the heading', () => {
-      const root = renderAddon({ addon: result });
+      const { heading, ...addon } = result;
+      const root = renderAddon({ addon, heading });
```

Comment:
I think if you're showing the heading as passed in props it should override the default so it's more clear otherwise, can you just leave that out here?

Reply:
I actually fixed the expected value :)